### ISSUE TITLE
Add backend routes for Models

### ISSUE THREAD

**Issue by nbarlowATI** [alan-turing-institute/DTBase#81]
Add backend endpoints for 
 - [x] adding Model Measures
 - [x] adding Models
 - [ ] configuring Models
 - [ ] running Models
 - [x] uploading Model results

**Comment by mhauru** [comment]
Waiting for #182 before starting work on this

**Comment by mhauru** [comment]
Superseded by the more granular ones #191 and #193
